swift/Samples/ModalContainer/ModalContainer.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name         = 'ModalContainer'
+  s.version      = '1.0.0.LOCAL'
+  s.summary      = 'See the README.'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { git: 'Not Published', tag: "podify/#{s.version}" }
+
+  # 1.7 is needed for `swift_versions` support
+  s.cocoapods_version = '>= 1.7.0.beta.1'
```

Comment:
The most recent version published is 1.9 or higher, so this can go away, I think

Reply:
Looks like all the podspec files in worklow have the same verson i.e. 1.7.0.beta.1. Should this be changed for all?

Reply:
Yes, we should update! Let's do it in an independent PR.

Reply:
It's worth keeping something in `s.cocoapods_version` in case  someone tries to pull in workflow while using an old CocoaPods installation, but it definitely doesn't need to be a beta. `1.7.0` should work

Reply:
Plus 1 to @amorde's comment